src/Resources/js/modules/participation-pre-toggle-handler.ts

```diff
@@ -41,6 +56,22 @@ export class ParticipationPreToggleHandler {
             && 0 < $checkbox.closest('.meal').find('.variation-row .text-variation').length; // has variations
     }
 
+    private isCombinedMealParticipation($checkbox: JQuery): boolean {
+        return 0 < $checkbox.closest('.meal-row.combined-meal').length;
```

Comment:
It is similar to line 55, isn't it? Maybe we make one method for it?

Reply:
I changed it. Is it still open?

Reply:
It's fine, could be resolved.